docs/ibc/light-clients/localhost/state-verification.md

```diff
@@ -0,0 +1,11 @@
+<!--
+order: 5
+-->
+
+# State verification
```

Comment:
When checking the feature branch to see how this works, I saw the `SentinelProof` concept. Should we elaborate on that here and how it's constructed?